utils/utils.jsx

```diff
@@ -765,6 +765,8 @@ export function applyTheme(theme) {
         changeCss('.app__body .emoji-picker .nav-tabs li a', 'fill:' + theme.centerChannelColor);
         changeCss('.app__body .post .post-collapse__show-more-button', `border-color:${changeOpacity(theme.centerChannelColor, 0.1)}`);
         changeCss('.app__body .post .post-collapse__show-more-line', `background-color:${changeOpacity(theme.centerChannelColor, 0.1)}`);
+        changeCss('.dropdown-menu > li > a', `color:${theme.centerChannelColor}`);
```

Comment:
These rules be prefixed by `.app__body` so that they only affect parts of the app that are themed and don't affect places like the system console where themes don't apply.
```suggestion
        changeCss('.app__body .dropdown-menu > li > a', `color:${theme.centerChannelColor}`);
```